sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/LDEPFSettings.java

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2023 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.commons.dto;
+
+import java.util.List;
+
+/**
+ * Represents a list of LDEPF settings
+ * This is a functional interface whose functional method is getLDEPFSettings
+ * the type of the input to the operation
+ *
+ * @param <T> the type of the result of the LDEPFSettings
+ *
+ * @see org.lfenergy.compas.sct.commons.util.SettingLDEPFCsvHelper
+ */
+@FunctionalInterface
+public interface LDEPFSettings<T> {
+
+    /**
+     * This method provides list of LDEPF settings
+     */
+    List<T> getSettings();
```

Comment:
In my opinion, it is too soon to find the correct interface until the algorithm is implemented. We will probably have to refacto this interface when we implement the algorithm on the next issues.

Reply:
I don't think there is a link between the algorithm and the interface, the interface will be a parameter for the service method  to provide all the setting.